include/ck_tile/ops/fmha/pipeline/block_fmha_pipeline_qr_ks_vs_whole_k_prefetch_default_policy.hpp

```diff
@@ -0,0 +1,380 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/fmha/pipeline/block_fmha_pipeline_qx_ks_vs_custom_policy.hpp"
+
+namespace ck_tile {
+
+struct BlockFmhaPipelineQRKSVSWholeKPrefetchDefaultPolicy
+    : BlockFmhaPipelineQXKSVSCustomPolicy</* QLoadOnce = */ false,
+                                          /* AsyncCopy = */ false,
+                                          /* NumPrefetchK = */ -1,
+                                          /* NumPrefetchV = */ 2>
+{
+    static constexpr index_t NumPrefetchV = 2;
+
+    template <typename Problem>
+    CK_TILE_HOST_DEVICE static constexpr ck_tile::index_t IsPreloadWholeNextIterationK()
+    {
+        if constexpr(Problem::BlockFmhaShape::kM0 <= 64)
```

Comment:
(1) shouldn't the return type be bool? (2) just return (Problem::BlockFmhaShape::kM0 <= 64)

Reply:
Changed!